app/models/pet.rb

```diff
@@ -18,7 +18,7 @@ def applicant
   end
 
   def self.pets_with_applications
-    select(:name, :id).joins(:pet_applications)
+    select(:name, :id, :approved).joins(:pet_applications)
```

Comment:
add approved to criteria selected so it can be called in the index view for our conditional